src/php/utils/docker/php-fpm-healthcheck

```diff
@@ -50,7 +50,7 @@ export SCRIPT_FILENAME="/status"
 FCGI_CONNECT_DEFAULT="localhost:9000"
 
 # Required software
-command -v cgi-fcgi 1> /dev/null || { >&2 echo "Make sure fcgi is installed (i.e. apk add --no-cache fcgi). Aborting."; exit 4; }
+FCGI_CMD_PATH=$(command -v cgi-fcgi) || { >&2 echo "Make sure fcgi is installed (i.e. apk add --no-cache fcgi). Aborting."; exit 4; }
```

Comment:
what would be the expected output for `FCGI_CMD_PATH`? I'd assume would be something like `/usr/bin/cgi-fcgi`

Reply:
@agustingomes depends on the system, that's why we use command, but yes!